app/services/hyrax/thumbnail_path_service_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hyrax v5.0.0rc2 - use site defaults instead of app wide defaults
+module Hyrax
+  module ThumbnailPathServiceDecorator
+    def default_image
+      Site.instance.default_work_image&.url || super
+    end
+  end
+end
+
+Hyrax::ThumbnailPathService.singleton_class.send(:prepend, Hyrax::ThumbnailPathServiceDecorator)
```

Comment:
```suggestion
Hyrax::ThumbnailPathService.singleton_class.send(:prepend, Hyrax::WorkThumbnailPathServiceDecorator)
```

Reply:
Why the above change?  We can get rid of another module and reduce duplication of knowledge.

Reply:
Hmm, are you suggesting to remove the `app/services/hyrax/thumbnail_path_service_decorator.rb` and put `Hyrax::ThumbnailPathService.singleton_class.send(:prepend, Hyrax::WorkThumbnailPathServiceDecorator)` in `app/services/hyrax/work_thumbnail_path_service_decorator.rb`?

Reply:
Yeah.

Reply:
I think my concern with that is the coupling of both of services.  In the `super` Hyrax actually has two different returns

`WorkThumbnailPathService` => `ActionController::Base.helpers.image_path 'work.png'`
and
`ThumbnailPathService` => `ActionController::Base.helpers.image_path 'default.png'`

If in an event that we want to change Hyku's thumbnail path and work thumbnail path then it would be easier to find and already set up.  But I'm not holding on to this idea too strongly.

Reply:
I'm alright with what you have.  But consider, your tests show what `super` means and the code looks like duplication (because it is; it just overlays different things, which is the nature of polymorphism/encapsulation)